### PR TITLE
Address lack of coverage for vpc.go

### PR DESCRIPTION
**Pull request by sayboras**
### Description
To address the lack of test coverage for `pkg/vpc/vpc.go` without any change in implementation. This is related to https://github.com/weaveworks/eksctl/pull/1477.

The current coverage of this file is improved 7.7% to 55.8%.

Noted: Some refractor needs to be done so that we can fully unit test other methods.

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, and `examples` directory)
- [ ] Manually tested
- [ ] Added labels for change area (e.g. `area/nodegroup`) and target version (e.g. `version/0.12.0`)
- [ ] Added note in `docs/release_notes/draft.md` (or relevant release note)

### CODE REVIEW

File: pkg/vpc/vpc_test.go [hunk 1]
```diff
@@ -14,7 +19,27 @@ import (
 	"github.com/weaveworks/eksctl/pkg/testutils/mockprovider"
 )
 
+type setSubnetsCase struct {
+	Cfg   *api.ClusterConfig
```
Comment (martina-if) [inline pkg/vpc/vpc_test.go]: Can we make these test fields private?

File: pkg/vpc/vpc_test.go [hunk 2]
```diff
@@ -55,40 +352,68 @@ var _ = Describe("VPC Endpoints", func() {
...
+			ClusterName: "notFoundCluster",
+			Private:     false,
+			Public:      false,
+			DCOutput:    nil,
+			Error:       nil,
+		}),
 	)
 })
+
+func stringPointer(s string) *string {
```
Comment (martina-if) [inline pkg/vpc/vpc_test.go]: For this functionality you can use `pkg/utils/strings/strings.go:Pointer()`

Comment (sayboras) [reply pkg/vpc/vpc_test.go]: oh cool, I was trying to find the existing function for the same. Thanks for pointing out.

File: pkg/vpc/vpc_test.go [hunk 3]
```diff
@@ -25,21 +26,21 @@ type setSubnetsCase struct {
 }
 
 type importVPCCase struct {
-	Cfg               *api.ClusterConfig
+	cfg               *api.ClusterConfig
 	ID                string
```
Comment (martina-if) [inline pkg/vpc/vpc_test.go]: Oh, I meant all the fields in these test structs: ID, DescripveVPCOutput, etc. As far as I see these are used only in this file right?

Comment (sayboras) [reply pkg/vpc/vpc_test.go]: oh yeah, I was thinking about that after pushed changes, let me make another commit.

Comment (sayboras) [reply pkg/vpc/vpc_test.go]: another look if you don't mind, thanks :)

**Pull request merged**